### DISCUSSION TITLE
외부 URL 및 동영상 플레이 설정

### DISCUSSION THREAD

**Discussion by Astoration** [naverz/zepeto-studio-kor#865] [5. Q&A]
안녕하세요
일반 월드에서는 외부 URL 오픈 및 동영상 플레이등이 제약되어있는걸로 알고 있는데요.
제페토측과 라이센스 협약을 진행하면 (공식월드 또는 공식월드에 준한) 이용가능한걸로 알고 있어서
현재 제페토측에서 저희 사업부 쪽으로 라이센스 진행중인걸로 알고있는데, 별도로 공식적인 안내는 없어서 일반 유니티 구현과 동일하게 구현하면 될까요?
(Application.OpenURL 등)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 공식 제휴맵들은 외부 URL이나 동영상 기능을 제공한 사례가 있습니다.

하지만 유니티를 사용한 월드에서는 아직 내부적으로 고려할 부분들이 많아서 해당 기능을 제공한 맵이 없습니다.
따라서 현재로서는 관련 안내가 존재하지 않고 출시가 불가한 점 양해 부탁 드립니다. 
가이드라인 참고 부탁 드립니다.
https://github.com/naverz/zepeto-studio-kor/discussions/595

감사합니다.
